src/gqlSchema.json

```diff
@@ -6050,6 +6122,18 @@
         "name": "__Schema",
         "description": "A GraphQL Schema defines the capabilities of a GraphQL server. It exposes all available types and directives on the server, as well as the entry points for query, mutation, and subscription operations.",
         "fields": [
+          {
+            "name": "description",
```

Comment:
Det ser ut som om du har brukt en nyere nodejs eller nyere apollo lokalt enn det som er spesifisert i package.json. Har du testa at dette fungerer?

Reply:
oj, jeg var ikke klar over at jeg kjører nyere versjon, ting fungerte når jeg testet lokalt med lokal-converter og graphql. Burde jeg teste noe spesifikt for å se om det ikke knekker?

Reply:
Veit ikkje. Fullt mulig det er uproblematisk. Har berre sett at nyere apollo legger til meir data i skjemaet.